ui/component/viewers/videoViewer/index.js

```diff
@@ -34,12 +34,38 @@ const select = (state, props) => {
   const playingUri = selectPlayingUri(state);
   const collectionId = urlParams.get(COLLECTIONS_CONSTS.COLLECTION_ID) || (playingUri && playingUri.collectionId);
   const isMarkdownOrComment = playingUri && (playingUri.source === 'markdown' || playingUri.source === 'comment');
+  const isClaimPlayable = (uri) => {
+    const claim = makeSelectClaimForUri(uri)(state);
+    // $FlowFixMe
+    return (
+      claim &&
+      // $FlowFixMe
+      claim.value &&
+      // $FlowFixMe
+      claim.value.stream_type &&
+      // $FlowFixMe
+      (claim.value.stream_type === 'audio' || claim.value.stream_type === 'video')
+    );
+  };
+  const nextUriInCollection = (fromUri) => {
+    return makeSelectNextUrlForCollectionAndUrl(collectionId, fromUri)(state);
+  };
+  const prevUriInCollection = (fromUri) => {
+    return makeSelectPreviousUrlForCollectionAndUrl(collectionId, fromUri)(state);
+  };
 
   let nextRecommendedUri;
   let previousListUri;
   if (collectionId) {
-    nextRecommendedUri = makeSelectNextUrlForCollectionAndUrl(collectionId, uri)(state);
```

Comment:
I think we should do a `makeSelectNextPlayableUrl..` etc. Keep the logic in the same place. What do you think?